packages/arb-token-bridge-ui/src/hooks/useNetworksAndSigners.tsx

```diff
@@ -96,7 +105,12 @@ export type NetworksAndSignersProviderProps = {
    *
    * @see https://reactjs.org/docs/render-props.html
    */
-  fallback: (status: UseNetworksAndSignersLoadingOrErrorStatus) => JSX.Element
+  fallback: (
+    status:
+      | UseNetworksAndSignersLoadingOrErrorStatus
+      | UseNetworksAndSignersNotSupportedStatus,
+    chainId?: number
+  ) => JSX.Element
```

Comment:
This would be a bit more precise for types:

```suggestion
type FallbackProps = 
  | { status: UseNetworksAndSignersLoadingOrErrorStatus }
  | { status: UseNetworksAndSignersNotSupportedStatus, chainId: number }
  
fallback: (props: FallbackProps) => JSX.Element
```